dash-pipeline/bmv2/dash_pipeline.p4

```diff
@@ -372,12 +372,31 @@ control dash_ingress(
         }
     }
 
-    apply {
-
-        /* Send packet on same port it arrived (echo) by default */
+// #ifdef UNDERLAY
 #ifdef TARGET_BMV2_V1MODEL
-        standard_metadata.egress_spec = standard_metadata.ingress_port;
+    action set_nhop(bit<9> next_hop_id) {
+        standard_metadata.egress_spec = next_hop_id;
+    }
+
+    @name("route|route")
+    // @Sai[skipHeaderGen=true]
+    table nhop {
+        key = {
+            meta.dst_ip_addr : lpm @name("meta.dst_ip_addr:destination");
+        }
+
+        actions = {
+            set_nhop();
+            @defaultonly deny;
```

Comment:
this is a philosophical question:
correct behavior is: no route drop the packet but if we do that all existing tests will fail and will have to be updated and may not even be possible to just add a 0.0.0.0/0 $PORT_0
incorrect will be to keep current behavior of reflecting back traffic when no route but this will help keep the existing cases still working without any change

no action here just something to keep in mind.

Reply:
Agreed and worthy of a comment in the code at least.

Reply:
I have changed the defaultonly to reflect the packets back to same port